src/Stryker.Core/Stryker.Core/Enumerations/LinqExpression.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stryker.Core.Enumerations
+{
+    /// <summary>
+    ///     Enumeration for the different kinds of linq expressions
+    /// </summary>
+    public enum LinqExpression
```

Comment:
Do we need a global enumeration for this? I think it's only used inside the LinqMutator. So it could be placed inside the LinqMutator file or in the same folder to keep the folder structure clean.

Reply:
I'll move it to the LinqMutator.